test/other/metadce/test_metadce_cxx_ctors1.sent

```diff
@@ -1,11 +1,11 @@
-__cxa_throw
-abort
-emscripten_memcpy_js
-emscripten_resize_heap
-environ_get
-environ_sizes_get
-fd_close
-fd_read
-fd_seek
-fd_write
-strftime_l
+/**@export*/__cxa_throw
```

Comment:
Can you have `run_metadce_test` strip these out to avoid this change?

Reply:
We can, but I wonder if it's worth it? It would add to test complexity, and actually I'd like to know if we change this comment in any way (though other tests would catch it presumably).

Reply:
There is already a bunch of logic in `run_metadce_test` for fishing out these symbol names.  Adding another line to the logic is worth it (IMHO) in order to keep these files are just flat symbol lists.

Reply:
For example, we already jump through hoops to strip out any stuff like comma and quote marks.

Reply:
Oh, thanks, I didn't realize we already did filtering here. sgtm, done.